packages/evolution-common/src/services/baseObjects/Segment.ts

```diff
@@ -120,6 +120,18 @@ export class Segment implements IValidatable {
         this.drivingCalculatedRoutings = ConstructorUtils.initializeComposedArrayAttributes(params.drivingCalculatedRoutings, Routing.unserialize);
     }
 
+    /**
+     * Checks if the segment is a transit segment
+     * @returns True if the segment is a transit segment, false otherwise
+     */
+    isTransit(): Optional<boolean> {
+        return this.modeCategory === 'transit';
+    }
+
+    get modeCategory(): Optional<SAttr.ModeCategory> {
+        return this.mode ? SAttr.mapModeToModeCategory[this.mode] as SAttr.ModeCategory : undefined;
```

Comment:
Il y a pas de danger que ça clash avec `this._attributes.modeCategory`? Faudrait-il enlever cet attribut? Ou retourner d'abord cet attribut avant de faire un map du mode?

Reply:
effectivement, on pourrait jsute enlever l'attribut.

Reply:
retiré